yarn-project/prover-client/src/orchestrator/orchestrator_mixed_blocks_2.test.ts

```diff
@@ -45,21 +51,26 @@ describe('prover/orchestrator', () => {
       acvmBinaryPath: acvmConfig?.expectedAcvmPath,
     });
     prover = new TestCircuitProver(simulationProvider);
+    queue = new MemoryProvingQueue();
+    agents = [new CircuitProverAgent(prover, 10, '1'), new CircuitProverAgent(prover, 10, '2')];
 
     builderDb = await MerkleTrees.new(openTmpStore()).then(t => t.asLatest());
     expectsDb = await MerkleTrees.new(openTmpStore()).then(t => t.asLatest());
-    builder = new ProvingOrchestrator(builderDb, prover, 1);
+    builder = new ProvingOrchestrator(builderDb, queue);
   }, 20_000);
 
-  describe('blocks', () => {
-    beforeEach(async () => {
-      builder = await ProvingOrchestrator.new(builderDb, prover);
```

Comment:
These tests were running with the Orchestrator's default concurrency limit of 64!